content/sensu-go/5.4/reference/checks.md

```diff
@@ -93,7 +72,221 @@ At every execution of a check command – regardless of success or failure – t
+
+For example, for three agents configured with the `system` subscription (agents A, B, and C), a check configured with the `system` subscription and `round_robin` set to `true` results in one monitoring event per interval, with the agent creating the event following the pattern A -> B -> C -> A -> B -> C for the first six intervals.
+
+_PRO TIP: You can use round-robin to distribute check execution workload across multiple agents when using [proxy checks with the proxy requests attributes](#using-a-proxy-check-to-monitor-a-multiple-proxy-entities)._
+
+### Scheduling
+
+You can schedule checks using the `interval`, `cron`, and `publish` attributes.
+Sensu requires that checks include either an `interval` attribute (interval scheduling), a `cron` attribute (cron scheduling), or the `publish` attribute set to `false` (ad-hoc scheduling).
+
+| check attributes | interval scheduling | cron scheduling | ad-hoc scheduling
+| --- | --- | --- | --- |
+| `interval` | ✅ | ❌ | ❌
+| `cron` | ❌ | ✅ | ❌
+| `publish` | `true` | `true` | `false`
+
+#### Interval scheduling
+
+You can schedule a check to be executed at regular intervals using the `interval` and `publish` check attributes.
+For example, to schedule a check to execute every 60 seconds, set the `interval` attribute to `60` and the `publish` attribute to `true`.
+
+_NOTE: When creating an interval check, Sensu calculates an initial offset to splay the check's
+first scheduled request.
+This helps to balance the load of both the backend and the agent, and may result in a delay before initial check execution._
+
+**Example interval check**
+
+{{< highlight json >}}
+{
+  "type": "CheckConfig",
+  "api_version": "core/v2",
+  "metadata": {
+    "name": "interval_check",
+    "namespace": "default"
+  },
+  "spec": {
+    "command": "check-cpu.sh -w 75 -c 90",
+    "subscriptions": ["system"],
+    "handlers": ["slack"],
+    "interval": 60,
+    "publish": true
+  }
+}
+{{< /highlight >}}
+
+#### Cron scheduling
+
+You can also schedule checks using [cron syntax][14].
+For example, to schedule a check to execute once a minute at the start of the minute, set the `cron` attribute to `* * * * *` and the `publish` attribute to `true`.
+
+**Example cron check**
+
+{{< highlight json >}}
+{
+  "type": "CheckConfig",
+  "api_version": "core/v2",
+  "metadata": {
+    "name": "cron_check",
+    "namespace": "default"
+  },
+  "spec": {
+    "command": "check-cpu.sh -w 75 -c 90",
+    "subscriptions": ["system"],
+    "handlers": ["slack"],
+    "cron": "* * * * *",
+    "publish": true
+  }
+}
+{{< /highlight >}}
+
+#### Ad-hoc scheduling
+
+In addition to automatic execution, you can create checks to be scheduled manually using the [checks API](../../api/check#the-checkscheckexecute-api-endpoint).
+To create a check with ad-hoc scheduling, set the `publish` attribute to `false` in addition to an `interval` or `cron` schedule.
```

Comment:
Maybe the link to the API is enough, but do we want an example of that adhoc creation or some English around "here's the definition, but you need to see the API documentation to know how to use it?"

Reply:
That's a great point. I think because this page is already so long, we should link out to the API docs here.